FogCreek/Glitch-Community
# Obtainable archaeology: fix addProjectToEmptyTeam styles

## Links
* Remix link: https://obtainable-archaeology.glitch.me

## GIF/Screenshots:
![Screen Shot 2019-07-03 at 2 55 57 PM](https://user-images.githubusercontent.com/938722/60617923-c660b480-9da2-11e9-98f0-a6c72654b997.png)

## Changes:
* modularize addProjectToEmptyTeam styles
* fix margin
* remove unused global styles

## How To Test:
* log into https://obtainable-archaeology.glitch.me
* create a new team
* verify the spacing is correct between the Add Project button and "Add projects to share with your team" banner
